packages/packherd-require/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@packages/packherd-require",
+  "version": "0.0.0-development",
+  "description": "Loads modules that have been bundled by `@tooling/packherd`.",
+  "private": true,
+  "main": "dist/require.js",
+  "scripts": {
+    "build": "tsc",
+    "build-prod": "yarn build",
+    "check-ts": "tsc --noEmit && yarn -s tslint",
+    "clean-deps": "rimraf node_modules",
+    "clean": "rimraf dist",
+    "test": "yarn test-unit",
+    "test-unit": "mocha -r @packages/ts/register --config ./test/.mocharc.js --reporter mocha-multi-reporters --reporter-options configFile=../../mocha-reporter-config.json",
+    "tslint": "tslint --config ../ts/tslint.json --project .",
+    "watch": "tsc --watch"
+  },
+  "dependencies": {
+    "convert-source-map": "^1.7.0",
+    "debug": "^4.3.1",
+    "esbuild": "^0.15.3",
+    "source-map-js": "^0.6.2"
+  },
+  "devDependencies": {
+    "@packages/ts": "0.0.0-development",
+    "mocha": "7.0.1",
+    "spok": "^1.3.0"
```

Comment:
Doesn't look like `spok` is used by `packherd-require` after the split.

Reply:
Removed: [`a6f143f` (#23257)](https://github.com/cypress-io/cypress/pull/23257/commits/a6f143f6b012f6fc2fb6c7670d7ad851d5f08fd0)